ccpp/driver/GFS_diagnostics.F90

```diff
@@ -1823,12 +1824,11 @@ subroutine GFS_externaldiag_populate (ExtDiag, Model, Statein, Stateout, Sfcprop
 
     idx = idx + 1
     ExtDiag(idx)%axes = 2
-    ExtDiag(idx)%name = 'totsnwb_ave'
-    ExtDiag(idx)%desc = 'bucket surface snow precipitation rate'
-    ExtDiag(idx)%unit = 'kg/m**2/s'
+    ExtDiag(idx)%name = 'totsnwb'
```

Comment:
We are using those fields for GFS post products for downstream users, if you need new total snow precipitation, please add a new variable, please do not replace the current fields.

Reply:
@ericaligo-NOAA I only removed time-averaging of buckets, as it seems confusing. Are you sure you want to keep time-averaging and existing description for precipitation buckets?

Reply:
@ericaligo-NOAA @junwang-noaa  Are we going to keep time-averaging of precipitation buckets? It does not make sense to me.

Reply:
It's interesting. But, yes, the global model is using these time-averaging of precipitation buckets and most operational centers do. So please don't remove/replace the existing fields. @WenMeng-NOAA

Reply:
Done